labtool2.0/src/components/pages/Nav.css

```diff
@@ -1,3 +1,10 @@
 .activeItem {
-  font-weight: bolder;
-}
\ No newline at end of file
+    font-weight: bolder;
+}
+.ui.inverted.pointing.menu .active.item {
+    border-bottom: 3px solid #fff;
+    border-radius: 3px;
+}
+.ui.inverted.pointing.menu .active.item::after {
+    display: none !important;
+}
```

Comment:
Custom CSS class name (instead of Semantic UI) should be used to apply these styles, as otherwise all Semantic UI menus will have these styles.

Reply:
Yeah, even if those are not currently in use. In the future they might be.